questions.md

```diff
@@ -0,0 +1,4 @@
+# Questions about pull requests
+
+* When we've used git in the past for classes, we've given instructors privileges to read each student assignment repository directly.  Commiting and pushing to GitHub is essentially assignment submission.  Is there a benefit to having students use submit a pull request when they are ready to submit instead?
```

Comment:
👋 Hi @cgwiseman ! Thank you for the question. 

I believe the major benefit to using a Pull Request (PR) for assignment submission is so that educators have a simple way to view file changes and like I'm doing here, provide comments/feedback line by line on the students work. 

This process also mimics what folks do out in industry, which would be beneficial in the sense that students could start to use Pull Requests as an effective way to collaborate.

If you happen to have any additional questions please let me know.

Cheers,

Eric